README.md

```diff
@@ -59,6 +59,32 @@ make build
 This command executes the Scarb build process, resulting in the creation of a
 Sierra program.
 
+## Test Project
+
+Run the following command to run all tests:
+
+```bash
+make test
+```
+
+If you wish to run tests from a specific module, use the MOD parameter. For instance, to run tests within the Mint module, execute:
+
+```bash
+make test MOD=Mint
+```
+
+To focus on a specific test function, utilize the FN parameter. For instance, to run only the "test_invalid_max_tick" function:
+
+```bash
+make test FN=test_invalid_max_tick
```

Comment:
This test doesn't work. After running it I get the following error:
`scarb test | filter by test_invalid_max_tick
     Running cairo-test yas with filter=test_invalid_max_tick
testing yas ...

grep: invalid option -- P
usage: grep [-abcdDEFGHhIiJLlMmnOopqRSsUVvwXxZz] [-A num] [-B num] [-C[num]]
        [-e pattern] [-f file] [--binary-files=value] [--color=when]
        [--context[=num]] [--directories=action] [--label] [--line-buffered]
        [--null] [pattern] [file ...]
thread 'main' panicked at 'failed printing to stdout: Broken pipe (os error 32)', library/std/src/io/stdio.rs:1019:9
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace`